### PR TITLE
Update TFX cicd lab to Vertex

### PR DESCRIPTION
**Pull request by BenoitDherin**
This PR updates the new TFX pipeline running on tfx==1.4.0 to be deployed on Vertex in CI/CD. 

One CloudBuild is triggered it will
- build the TFX pipeline container
- compile the TFX pipeline into a JSON description
- deploy and run the TFX pipeline on Vertex

The build will only complete successfully after the pipeline run completes successfully.

Please test and review!

### CODE REVIEW

**Comment by dougkelly** [comment]
1. Successful replication of pipeline run in 1 hour 10 min for **Exercise: manually trigger CI/CD pipeline run with Cloud Build**: 

Screenshot 2021-12-10 3 44 38 PM

**Comment by dougkelly** [comment]
2. Having trouble with **Exercise: Setting up GitHub integration* triggering a successful build from Github tag or from Cloud Build manual run in console:

Screenshot 2021-12-10 4 44 27 PM

File: notebooks/tfx_pipelines/cicd/solutions/tfx_cicd_vertex.ipynb [hunk 1]
```diff
@@ -0,0 +1,313 @@
+{
```
Comment (dougkelly) [inline notebooks/tfx_pipelines/cicd/solutions/tfx_cicd_vertex.ipynb:1]: <p>Consider moving the line "Navigate to <a href="https://console.cloud.google.com/vertex-ai/pipelines" target="_blank">Vertex Pipelines</a> page in the Google Cloud Console." down to the end "Exercise: manually trigger CI/CD pipeline run with Cloud Build". This feels like we tell them to go to Vertex Pipelines without a specific action / purpose when they need to switch back to continue. Instead, at the end of this exercise, they will instead see the Cloud Build triggered pipeline run to check out.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (dougkelly) [inline notebooks/tfx_pipelines/cicd/solutions/tfx_cicd_vertex.ipynb:1]: <p>Suggestion: to help frame learner expectations on runtime, consider adding something like:</p><p> </p><p>"Note: building and deploying the container below is expected to take 10-15 min."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (dougkelly) [inline notebooks/tfx_pipelines/cicd/solutions/tfx_cicd_vertex.ipynb:1]: <p>Nit: Add space between GCPREGION so it reads YOUR GCP REGION</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (dougkelly) [reply notebooks/tfx_pipelines/cicd/solutions/tfx_cicd_vertex.ipynb:1]: Similarly, consider adding a note during the first manual trigger, something like:

"Note: this step will build, deploy, and start a pipeline run. The full pipeline run with take about 1 hour 10 min; please continue on to the next exercise."

Comment (BenoitDherin) [reply notebooks/tfx_pipelines/cicd/solutions/tfx_cicd_vertex.ipynb:1]: <p>Done!</p>

Comment (BenoitDherin) [reply notebooks/tfx_pipelines/cicd/solutions/tfx_cicd_vertex.ipynb:1]: <p>Done!</p>

Comment (BenoitDherin) [reply notebooks/tfx_pipelines/cicd/solutions/tfx_cicd_vertex.ipynb:1]: <p>Done!</p>

**Pull request merged**